test/transmuxer.test.js

```diff
@@ -2033,7 +2036,7 @@ QUnit.test('calculates baseMediaDecodeTime values relative to a customizable bas
 
   boxes = mp4.tools.inspect(segment);
   tfdt = boxes[0].boxes[1].boxes[1];
-  QUnit.equal(tfdt.baseMediaDecodeTime, 1324, 'calculated baseMediaDecodeTime');
+  QUnit.equal(tfdt.baseMediaDecodeTime, bMDTValue + 90, 'calculated baseMediaDecodeTime');
```

Comment:
The `+ 90` part is a little unclear to me. Maybe a comment about how that's calculated?